src/helpers/_media-queries.scss

```diff
@@ -13,10 +21,9 @@ $mq-show-breakpoints: ();
 // When building a stylesheet for IE8, set $mq-responsive to false in order to
 // 'rasterize' any media queries.
 
+$mq-responsive: true;
```

Comment:
Moving this because the way Sass variable scoping works means that unless `$mq-responsive` is defined before this point, the `$mq-responsive` declaration only works within the scope of the `@if` – which caused a lot of headaches in the tests.